flaml/tune/spark/__init__.py

```diff
@@ -0,0 +1,8 @@
+from flaml.tune.spark.utils import (
+    check_spark,
+    get_n_cpus,
+    with_parameters,
+    customize_learner,
```

Comment:
`customize_learner` is a concept in `automl`. There are two ways to deal with it.
1. Rename `customize_learner` into a more general concept, such as `broadcast_code` which is suitable for the `tune` layer.
2. Move `customize_learner` into `flaml.automl`.

I prefer 1 because even without doing automl, there can be cases where the broadcasting is needed when using `flaml.tune`.

Reply:
Refactored to broadcast_code. Thanks.